.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@vue/app"]
```

Comment:
you're missing the package import for this dependency

Reply:
Removing this dependency. Resolving for now.